js/fetch_data.js

```diff
@@ -0,0 +1,166 @@
+                                <h2 class="work-title-preview">${obj.title}</h2>
+                                <span class="close">&times;</span>
+                            </div>
+
+                            <div class="frame-2">
+                                <h3 class="frame-subtitle">${obj.subtitle}</h3>
+                                <div class="rounded-bullet"></div>
+                                <h3 class="position-title">${obj.position}</h3>
+                                <div class="rounded-bullet"></div>
+                                <h3 class="work-year">${obj.year}</h3>
+                            </div>
+                            <img src="img/${obj.img}" alt="portfolio project image" class="cards-img-preview">
+                            <!-- body content -->
+                            <div class="preview-body-content"> 
+                                <p class="skills-description-preview">
+                                    ${obj.description}
+                                </p>
+                                <div class="skills-and-footer-body">
+                                    <ul class="skills-ul-preview">
+                                        ${returnSkills(obj.skills)}
+                                    </ul>
+        
+                                    <div class="footer-popup">
+                                        <!-- customized buttons -->
+                                        <a href="${obj.live_version}" class="footer-popup-link">
+                                        <div class="footer-popup-button">
+                                            <h5 class="footer-button-title">See Live</h5>
+                                            <img src="img/github_icon_preview.svg" alt="">
+                                        </div>
+                                        </a>
+                                        <a href="${obj.source_code}" class="footer-popup-link">
+                                        <div class="footer-popup-button">
+                                            <h5 class="footer-button-title">See Source</h5>
+                                            <img src="img/go-live-icon.svg" alt="">
+                                        </div>
+                                        </a>
+                                    </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>`;
+
+  const modal = document.getElementById('modal');
+  modal.style.display = 'block';
+  // // Get the <span> element that closes the modal
+  const span = document.getElementsByClassName('close')[0];
+
+  // // When the user clicks on <span> (x), close the modal
+  span.onclick = () => {
+    modal.style.display = 'none';
+  };
+
+  window.onclick = (event) => {
+    if (event.target === document.getElementById('modal')) {
+      modal.style.display = 'none';
+    }
+  };
+}
```

Comment:
- [ ] [REQUIRED] Kindly ensure that the desktop modal displays the rest of the modal contents as provided in the figma design. Right now I can't see the contents, I can only see the modal image. I would suggest you place the content and the image in a single div.
- [ ] [REQUIRED] Please make sure that the modal for the mobile fits properly at the center. You currently have more margin at the top of the mobile modal.
- [ ] [REQUIRED] Also, please check the modal of the the mobile version and give it two buttons (See-live and See-source) their appropriate margins as it is in the figma.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="1005" alt="Screenshot 2022-11-17 at 20 29 22" src="https://user-images.githubusercontent.com/58692590/202543060-f2e3d06a-1378-4c8e-8958-1d96bbff745d.png"> | <img width="558" alt="Screenshot 2022-11-17 at 20 30 13" src="https://user-images.githubusercontent.com/58692590/202543086-d8687c08-62d6-41b2-afb7-ebda77815790.png">

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="429" alt="Screenshot 2022-11-17 at 20 28 32" src="https://user-images.githubusercontent.com/58692590/202543552-33c7d67b-3db1-4b23-9dc4-54669de18cda.png"> | <img width="299" alt="Screenshot 2022-11-17 at 20 27 46" src="https://user-images.githubusercontent.com/58692590/202543616-aa3d3368-d774-4d3e-8aa2-fed5b8df3ef5.png">

Reply:
Hello @chiztechnology , I recommended the above changes for you while you pushed a new commit. Your linters passed initially but just as I recommended changes, you pushed new commits at the same time, hence, your stylelint is now failing. Kindly fix this and all the recommended changes before requesting another review.
Happy coding 🍾

Reply:
- [ ] @chiztechnology great work implementing the changes requested however, I noticed the website broke after the change was implemented, I can only access one popup only once, it requires me to reload the page to view the next popup, kindly fix it.